web/src/api.ts

```diff
@@ -32,7 +32,7 @@ export interface QueryResultPayload {
 export async function getPackages({
   search = "",
   limit = 10,
-  cursor = null,
+  cursor = null as Cursor,
```

Comment:
This is quite unsettling... Why did we need this?

Reply:
I want to declare cursor as optional, the same with an interface:

```ts
interface IGetPackagesArgs {
  search: string;
  limit: number;
  cursor?: Cursor;
}

export async function getPackages({
  search = "",
  limit = 10,
  cursor,
}: IGetPackagesArgs): Promise<PackagesResponse> {
```

I can update it to use the interface, I like to keep the code shorter, but probably `null as Cursor` hard to read (even it took me a bit to remember what I wanted to do)

Reply:
Got it, thanks. I think it's okay to define the type of the argument on top of the function.

The `null as Cursor` can lead to problematic "valid" type check while the program crashes runtime.